ms2/test/src/org/labkey/test/tests/ms2/XTandemTest.java

```diff
@@ -144,19 +144,16 @@ private void verifyComparePeptides()
         uncheckCheckbox(Locator.name("exactMatch"));
         clickButton("Search");
         assertElementPresent(Locator.linkContainingText(SAMPLE_BASE_NAME + " (test2)"));
-        clickAndWait(Locator.id("expandCollapse-ProteinSearchProteinMatches"), 0);

```

Comment:
I'd like to leave this collapsed by default. Users typically want to see the results associated with a protein, so they don't care as much about the proteins themselves. However, if they don't see any results, it's useful to check the protein list to see what is eligible to match against.